main.tf

```diff
@@ -234,6 +238,7 @@ output "helm_values" {
     "placeholder_tls_cert_backup_aws_secret_key", var.tls_cert_backup_aws_secret_key),
     "placeholder_tls_cert_restore_aws_access_key", var.tls_cert_restore_aws_access_key),
     "placeholder_tls_cert_restore_aws_secret_key", var.tls_cert_restore_aws_secret_key),
+    "placeholder_vault_backup_s3_key_prefix", var.vault_backup_s3_key_prefix),
```

Comment:
See multy. I think it doesn't have a change that includes this: placeholder_vault_backup_s3_key_prefix ?

Reply:
nvm. I see it.